qtism/data/storage/xml/marshalling/Html5LayoutMarshaller.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ * Copyright (c) 2013-2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ * @author Jérôme Bogaerts <jerome@taotesting.com>
+ * @license GPLv2
+ */
+
+namespace qtism\data\storage\xml\marshalling;
+
+use DOMElement;
+use qtism\common\utils\Version;
+use qtism\data\content\FlowCollection;
+use qtism\data\content\xhtml\html5\Html5Element;
+use qtism\data\QtiComponent;
+use qtism\data\QtiComponentCollection;
+use qtism\data\storage\xml\versions\QtiVersion;
+
+/**
+ * The Marshaller implementation for object elements of the content model.
+ */
+class Html5LayoutMarshaller extends ContentMarshaller
```

Comment:
WDYT If we rename this class to Html5LayoutMarshaller -> Html5ItemBodyMarshaller

Reply:
Renamed to Html5ContentMarshaller